### ISSUE TITLE
Image source ignores Mediawiki image whitelists

### ISSUE THREAD

**Issue by labster** [tstaerk/adaptivethumb#4]
Hi, I'm a security reviewer for the wiki farm miraheze.org (@miraheze).  One of our wikis requested this extension, and I gave it a read through.  The main issue that I have is that the src attribute doesn't respect `$wgEnableImageWhitelist` or `$wgAllowExternalImages` or `$wgAllowExternalImagesFrom`.

Of course, we cannot legally use your code without a license, so we cannot install it.  But I thought you would like to be aware of the security issues.

**Comment by tstaerk** [comment]
I want my code to be used. You say I have to put it under a license for that?

**Comment by Reception123** [comment]
@tstaerk Yes, for us to use your code it needs a license. This is an example of a license: https://github.com/wikimedia/mediawiki-extensions-Translate/blob/master/COPYING and https://github.com/wikimedia/mediawiki-extensions-Translate/blob/master/Translate.php#L11

**Comment by tstaerk** [comment]
Thank you so much, I am not a lawyer, but now I understand why you need licenses. I added the one mentioned by your colleague.

Also, great review regarding $wg*

**Comment by tstaerk** [comment]
committed a change, now respecting $wgAllowExternalImages

**Comment by tstaerk** [comment]
Thank you.

I added a license and now it adheres to $wgAllowExternalImages

regards

Thorsten
